docs/installation/docker-compose.md

```diff
@@ -138,57 +132,140 @@ docker-compose up -d
+| Matomo      | [https://islandora.traefik.me/matomo/](https://islandora.traefik.me/matomo/)                   |
+| Code Editor | [https://islandora.traefik.me:8443/](https://islandora.traefik.me:8443/)                       |
 
-Direct a browser to [https://islandora.traefik.me/](https://islandora.traefik.me/). If your containers are still "spinning up", you will see a white screen with the words "Bad Gateway". This often lasts 2-5 minutes, and should be shorter for subsequent launches. If it takes more than a few minutes, check to make sure that none of your containers have failed to launch (see note above on "Troubleshooting - docker containers exit without warning"). When all containers are ready, you should see a basic Drupal login screen.
+If your containers are still "spinning up", you will see a white screen with the words "Bad Gateway". 
 
 !["Bad Gateway" white screen while still loading](../assets/docker_bad_gateway_still_loading.png)
 
+This often lasts 2-5 minutes, and should be shorter for subsequent launches. If it takes more than a few minutes, check to make sure that none of your containers have failed to launch using `docker ps -a`. When all containers are ready, you should see a basic Drupal login screen.
+
 ![Drupal login screen](../assets/docker_drupal_login_screen.png)
 
-To log in:
+By default, the credentials to login are
 
 * username: **admin**
 * password: **password**
 
-!!! Note Docker logs and Docker Compose logs
-    `docker logs -tf isle-dc_drupal_1` shows Docker logs for the "drupal" container, which will continue to have new log entries as long as the drupal box is still starting up. When it gets to `confd using 'env' backend`, you're done. `docker-compose logs` is like a firehose, showing the log messages from all containers.
+If one of your containers fails to launch, you'll need to check its logs using `docker-compose logs -tf SERVICENAME`.  For example, to see Drupal's logs, use `docker-compose logs -tf drupal`.  Usually, when you see `confd using 'env' backend`, the service has booted up successfully.  If you want to see all logs, just run `docker-compose logs`. 
 
-## Spinning down your Islandora site
+### Shutting down your Islandora site
 
 To shut down the containers without destroying your site, use `docker-compose down`. To also destroy your "state" (i.e. your content, your database, your files), use `docker-compose down -v`.
 
-## Editing Code in Docker
+### Bringing your Islandora site back up
 
-If you used `make local` then the drupal root folder is in a new directory in the isle-dc folder named `codebase`. This is live and editable in whatever development environment you would like.  If you just did `make demo`, you will need to spin down your containers with `-v` to destroy your state before starting a new one with `make local`.
+Use `docker-compose up -d` to bring a site back up. You should _not_ run `make local` or `make demo` again unless you used `-v` when shutting down the site.
```

Comment:
So, `make local` and `make demo` should only ever be run once, unless you delete the volumes, and if changes are made to any of the docker-compose files, we should only be using the `make -B docker-compose.yml` command, correct?